playgrounds/petclinic-maven-sb-2/src/main/resources/templates/welcome.html

```
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html xmlns:th="https://www.thymeleaf.org" th:replace="~{fragments/layout :: layout (~{::body},'home')}">
+
+<body>
+
+  <h2 th:text="#{welcome}">Welcome</h2>
+  <div class="row">
+    <div class="col-md-12">
+      <img class="img-responsive" src="../static/resources/images/pets.png" th:src="@{/resources/images/pets.png}" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)